website/docs/docs/concepts/10-simple-schema.md

```diff
@@ -109,6 +115,23 @@ labels: "map[string]string"
 metrics: "map[string]float"
 ```
 
+### Custom Types
+
+Custom types are specified in the separate `types` section.
+They provide a map of names to type specifications, that follow the simple schema.
+
+Example:
+
+```yaml
+schema:
+  types:
+    Person:
+      name: string
+      age: integer
+  spec:
+    people: '[]person | required=true`
```

Comment:
Does it need to match case sensitive it doesn't have to be `[]Person` upper case P?

Reply:
It does! Fixed.